### ISSUE TITLE
IE 11 bug

### ISSUE THREAD

**Issue by simonguo** [formatjs/react-intl#993]
Warning: Failed propType: Invalid prop `children` of type `object` supplied to `IntlProvider`, expected a single ReactElement. Check the render method of `App`.

**Comment by cwtuan** [comment]
it only happens in IE 11? Could you provide more detail of you App.js

**Comment by ghostd** [comment]
Are you sure you put only one child inside the `<IntlProvider />`?

**Comment by simonguo** [comment]
@ghostd  I'm sure

**Comment by papasmile** [comment]
Sample code would be good.

**Comment by 53c701d** [comment]
Same error. This occur in IE when children in PropTypes passed as element.
![wmxflqaktwi8zn4nzlzf8a](https://user-images.githubusercontent.com/11475496/33698518-ba4a32ba-db3f-11e7-9624-02c3fd7ca718.png)

**Comment by sorenhoyer** [comment]
Same here.

**Comment by MathMesquita** [comment]
Hey everyone, i don't know if you had fixed it yet, but i found [this awesome post](http://uidevdaily.com/2018/ie11-error-failed-prop-type-invalid-prop-children-of-type-object/) showing how to fix the problem.

#### TLDR

you must add `babel-polyfill` to your `webpack.config.js` entries before the `react` and `react-dom`

```
entry: [
  ...
  'babel-polyfill',
  'react',
  'react-dom',
  ...
]
```

#### CRA

In case you are using `create-react-app` just add `import 'babel-polyfill'` as the first import from your whole application(usually the first line of index.js)
